src/presentation/react/core/components/sync-summary/SyncSummary.tsx

```diff
@@ -186,10 +188,16 @@ const getOriginName = (source: PublicInstance | Store) => {
     }
 };
 
-const SyncSummary = ({ response, onClose }: SyncSummaryProps) => {
+const SyncSummary = ({ response, instance, onClose }: SyncSummaryProps) => {
     const { compositionRoot } = useAppContext();
+    const payload = response.payload;
     const classes = useStyles();
     const [results, setResults] = useState<SynchronizationResult[]>([]);
+    const downloadJSON = () => {
+        const date = moment().format("YYYYMMDDHHmm");
+        const fileName = `synchronization-${instance}-${date}.json`;
```

Comment:
Since we are adding a name that might not follow the ``kebabCase`` format.

```suggestion
        const fileName = _.kebabCase(`synchronization-${instance}-${date}.json`);
```